### PR TITLE
Add hasAlias method to ColorSpace and NamedTransform classes

### PR DESCRIPTION
**Pull request by annieln**
Issue #2047

### CODE REVIEW

File: src/OpenColorIO/NamedTransform.h [hunk 1]
```diff
@@ -10,6 +10,7 @@
 #include <OpenColorIO/OpenColorIO.h>
 
 #include "TokensManager.h"
+#include "Platform.h"
```
Comment (doug-walker) [inline src/OpenColorIO/NamedTransform.h:13]: Minor: Might be better moving this to the .cpp file.

File: src/OpenColorIO/ColorSpace.cpp [hunk 2]
```diff
@@ -147,6 +148,16 @@ const char * ColorSpace::getAlias(size_t idx) const noexcept
     return "";
 }
 
+bool ColorSpace::hasAlias(const char * alias) const noexcept
+{
+    for (size_t idx = 0; idx < getImpl()->m_aliases.size(); ++idx) {
```
Comment (doug-walker) [inline src/OpenColorIO/ColorSpace.cpp:153]: Very minor: The coding style for the project moves the opening bracket "{" to its own line.

**Review by doug-walker — CHANGES_REQUESTED** [review]
This is great work Annie!  Really nice job on the unit tests!

There is one other part necessary so that the new method shows up in the Python binding too.  Please update the files:
OpenColorIO/src/bindings/python/PyColorSpace.cpp
OpenColorIO/src/bindings/python/PyNamedTransform.cpp
OpenColorIO/tests/python/ColorSpaceTest.py
OpenColorIO/tests/python/NamedTransformTest.py

The syntax using for the Python bindings (pybind11) is a bit arcane, but you should be able to copy what you see for the other methods. Let us know if you have a question.

**Review by doug-walker — APPROVED** [review]
This is excellent work, thank you so much Annie!

File: tests/python/ColorSpaceTest.py [hunk 3]
```diff
@@ -356,7 +356,7 @@ def test_transform(self):
 
     def test_aliases(self):
         """
-        Test NamedTransform aliases.
+        Test ColorSpace aliases.
```
Comment (KelSolaar) [inline tests/python/ColorSpaceTest.py:359]: Nice catch!

**Review by KelSolaar — APPROVED** [review]
LGTM!

**Pull request merged**